src/app/component/auth/components/sign-in/sign-in.component.ts

```diff
@@ -82,28 +67,66 @@ export class SignInComponent implements OnInit, OnDestroy {
     this.userOwnSignIn.email = email;
     this.userOwnSignIn.password = password;
 
-    this.userOwnSignInService.signIn(this.userOwnSignIn).subscribe(
+    const subscription = this.userOwnSignInService.signIn(this.userOwnSignIn).subscribe(
       (data: UserSuccessSignIn) => {
         this.onSignInSuccess(data);
       },
       (errors: HttpErrorResponse) => {
         this.onSignInFailure(errors);
         this.loadingAnim = false;
       });
+    this.subscriptions = [...this.subscriptions, subscription];
```

Comment:
looks good .. but can you look at it and maybe this pattern will be a little better (not sure)

@Component({...})
export class SomeCoolName implements OnInit, OnDestroy {
    
    subscription: Subscription
    ngOnInit () {
        var observable1$ = cool1();
        var observable2$ = cool2();
        var subscription1$ = observable.subscribe(x => console.log("woow"));
        var subscription2$ = observable.subscribe(x => console.log("woow"));
        this.subscription.add(subscription1$)
        this.subscription.add(subscription2$)
    }
    ngOnDestroy() {
        this.subscription.unsubscribe()
    }
}